test/widget_test.dart

```diff
@@ -5,26 +5,10 @@
 // gestures. You can also use WidgetTester to find child widgets in the widget
 // tree, read text, and verify that the values of widget properties are correct.
 
-import 'package:flutter/material.dart';
-import 'package:flutter_test/flutter_test.dart';
+// import 'package:flutter/material.dart';
```

Comment:
I'm definately no expert on flutter, but it might be a good idea just to delete these rayther than leave them in commented out.  We can always get the history back out of git if we need :-)

Reply:
Actually need flutter_test very soon to pass CI testing.

Reply:
I commented it out because it was causing some build problems and I wasn't using it.  But I knew that someone might need it.

Reply:
That will be because there are no tests yet, but that sall good. Leave it for now 👍I can fix later and add at least one test so it gets past build on codemagic.

Reply:
Yes lets please not leave any code, even tests commented out. We'll start adding tests asap